### ISSUE TITLE
Link to gg.deals

### ISSUE THREAD

**Issue by Revadike** [bartervg/barter.vg#247]
# What problem does this feature address?
There is a lack of linking to gg.deals, a game deal comparison site, like ITAD (but better)

## Describe a solution
Link games to their gg.deals page, similar to ITAD.

## Examples of similar features
ITAD linking

**Comment by antigravities** [comment]
Worth noting is that gg.deals advertises and has referral relationships with grey market resellers (albeit with a warning).

**Comment by Revadike** [comment]
True, though its pretty obvious and not misleading. It's free too, so I doubt many care.

**Comment by lillje** [comment]
We try to not promote / support anything that contains a referral link. As far as I know, ITAD is an exception because a lot of people rely on it.

**Comment by Revadike** [comment]
Rely on what?

**Comment by bartervg** [comment]
> 
> 
> Rely on what?

Rely on ITAD.

The technical obstacle is that, like ITAD, gg.deals uses strings rather than appIDs. Do they have an API like ITAD that converts appIDs into strings?

**Comment by Revadike** [comment]
This technical obstacle may be resolved in the future:
https://github.com/ggdeals/placeholder/issues/1#issuecomment-807494079

**Comment by Revadike** [comment]
Now possible!
https://github.com/ggdeals/placeholder/issues/1#issuecomment-822379587

**Comment by Revadike** [comment]
Thanks for adding this! /closing
![image](https://user-images.githubusercontent.com/4411977/117079214-a3efa780-ad3b-11eb-9368-56426cbbc2ea.png)

**Comment by bartervg** [comment]
> 
> 
> Thanks for adding this! /closing
> ![image](https://user-images.githubusercontent.com/4411977/117079214-a3efa780-ad3b-11eb-9368-56426cbbc2ea.png)

It's a work-in-progress. I wouldn't have closed it yet.

**Comment by Revadike** [comment]
What's still WiP?

**Comment by Revadike** [comment]
@bartervg Please release this to the public. It's currently only enabled on my account (and it works great).

**Comment by lillje** [comment]
> 

As I said before:
We try to not promote / support anything that contains a referral link. ITAD is enough of that.

It was never a matter of this functionality not working. Adding gg.deals for all users would result in associating Barter.vg with the grey market, which we wanted to avoid so far.

It was enabled on your account only, as a special favor to you.

**Comment by Revadike** [comment]
> > 
> 
> As I said before: We try to not promote / support anything that contains a referral link. ITAD is enough of that.
> 
> It was never a matter of this functionality not working. Adding gg.deals for all users would result in associating Barter.vg with the grey market, which we wanted to avoid so far.
> 
> It was enabled on your account only, as a special favor to you.

🤨
